dvc/repo/__init__.py

```diff
@@ -80,7 +80,14 @@ class Repo:
     from dvc.repo.status import status
     from dvc.repo.update import update
 
-    def __init__(self, root_dir=None, scm=None, rev=None, subrepos=False):
+    def __init__(
+        self,
+        root_dir=None,
+        scm=None,
+        rev=None,
+        subrepos=False,
+        uninitialized=False,
```

Comment:
We might consider getting rid of this in the future (for now needed for backward compatibility). Currently `dvc init` doesn't do much useful things and all `.dvc` contents could be (and should probably) be created dynamically.

Even things like `dvc add` could potentially just work in a pure git repo. Just something to think about...